[javascript/en] fixed typo (#4395)
adambard/learnxinyminutes-docs: javascript.html.markdown

// Get subarray of elements from index 1 (include) to 4 (exclude)
myArray0.slice(1,4); // = [false,"js",12]

> // Remove 4 elements starting from index 2, and insert their strings

Reviewer: ```suggestion
// Remove 4 elements starting from index 2, and add instead the strings
```

Reviewer: It doesn't look like a typo to me. It means to insert "there" the strings (details [here](https://www.w3schools.com/jsref/jsref_splice.asp)). However, I agree it is not very clear. I made a suggestion, though I believe the whole section could benefit from a rewrite.
